javascript/ql/lib/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -826,7 +826,7 @@ module NodeJSLib {
    * for example `http.request(url)`.
    */
   class NodeJSClientRequest extends ClientRequest {
-    override NodeJSClientRequest::Range self;
+    NodeJSClientRequest() { this instanceof NodeJSClientRequest::Range }
```

Comment:
Is there a reason we can't use `instanceof` in the base type here?

Reply:
I suppose not.  
`ClientRequest` is already instance of the `::Range` pattern, so I suppose I didn't want to introduce "nested" `::Range` patterns.  
But it should just work, so I've changed it.